### PR TITLE
Script element

### PR DESCRIPTION
**Pull request by juansnow01**
Added section to show what internal script element would look like

#### Summary

Added a small section to show what added script into HTML would look like when it isnt linked to an external file.

#### Motivation

Even though it isnt recommended, people learning should still see what that would look like in the case they run into it in the future

#### Metadata

- [ ] Adds a new document
- [ x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md [hunk 1]
```diff
@@ -256,6 +256,12 @@ If done correctly, when you save your HTML and refresh your browser you should b
 
 - The JavaScript has added an empty list to the page. Now when you click anywhere outside the list, a dialog box will pop up asking you to enter some text for a new list item. When you press the OK button, a new list item will be added to the list containing the text. When you click on an existing list item, a dialog box will pop up allowing you to change the item's text.
 - The CSS has caused the background to go green, and the text to become bigger. It has also styled some of the content that the JavaScript has added to the page (the red bar with the black border is the styling the CSS has added to the JS-generated list.)
+> **Also:** JavaScript can be added in its own element without linking an external file, it is not highly recommended but still very possible. It would look something like this:
+```JavaScript 
+<script>
+document.getElementById("demo").innerHTML = "Hello JavaScript!";
+</script> 
+```
```
Comment (teoli2003) [inline files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:264]: Drive-by comment (not a review); this doesn't imply we should make this change at all.

```suggestion

JavaScript code can also be added without linking an external file, by using the `<script>` element. It is not recommended but possible and would look something like this:
```html 
<script>
  document.getElementById("demo").textContent = "Hello JavaScript!";
</script> 
```
```

Comment (estelle) [reply files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:264]: If you put that JS in the head it will fail, as #demo doesn't exist yet. You can't defer or async this type of inline script either.

Comment (estelle) [reply files/en-us/learn/html/introduction_to_html/the_head_metadata_in_html/index.md:264]: We either should not include this, or, if we include `<script>` in `<head>` it should be to advise/warn that neither `async` nor `defer` work, and that they likely need to wrap the whole thing in DOMContentLoaded, or whatever the best practice is and link to a tutorial.
